# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1522969310 25200
#      Thu Apr 05 16:01:50 2018 -0700
# Branch stable
# Node ID 2bc071fc7f6bf7a29a8301709a70908035a148dd
# Parent  8cffe74b75416b6574ce4b3f30d1243f48e4d4cd
pkg: do not error when archive file name differs from pkg name (bug #53561)

* pkg/private/install.m: Remove check and error when archive file name
differs from pkg name in DESCRIPTION file.

diff --git a/scripts/pkg/private/install.m b/scripts/pkg/private/install.m
--- a/scripts/pkg/private/install.m
+++ b/scripts/pkg/private/install.m
@@ -107,24 +107,16 @@ function install (files, handle_deps, pr
 
         ## Make sure the package contains necessary files.
         verify_directory (packdir);
 
         ## Read the DESCRIPTION file.
         filename = fullfile (packdir, "DESCRIPTION");
         desc = get_description (filename);
 
-        ## Verify that package name corresponds with filename.
-        [dummy, nm] = fileparts (tgz);
-        if ((length (nm) >= length (desc.name))
-            && ! strcmp (desc.name, nm(1:length (desc.name))))
-          error ("package name '%s' doesn't correspond to its filename '%s'",
-                 desc.name, nm);
-        endif
-
         ## Set default installation directory.
         desc.dir = fullfile (prefix, [desc.name "-" desc.version]);
 
         ## Set default architectire dependent installation directory.
         desc.archprefix = fullfile (archprefix, [desc.name "-" desc.version]);
 
         ## Save desc.
         descriptions{end+1} = desc;
